SDK/AppCenter/Microsoft.AppCenter.Shared/WrapperSdkInfo.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.AppCenter
+{
+    public class WrapperSdkInfo
+    {
+        public string WrapperSdkVersion { get; private set; }
```

Comment:
Ideally blank lines are added between fields, say if there was xml doc for it you would split them.